src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -275,6 +282,11 @@ public ResponseEntity<StudentExam> getStudentExamForConduction(@PathVariable Lon
         if (!user.getId().equals(studentExam.getUser().getId())) {
             throw new AccessForbiddenException("The requested exam does not belong to the requesting user");
         }
+
+        if (!Boolean.TRUE.equals(studentExam.isStarted())) {
```

Comment:
Can this even be `null`?
```suggestion
        if (!studentExam.isStarted()) {
```

Reply:
Yes, it can.